src/Symfony/Component/Mime/Tests/RawMessageTest.php

```diff
@@ -32,4 +32,18 @@ public function testToString()
         $this->assertEquals('some string', $message->toString());
         $this->assertEquals('some string', implode('', iterator_to_array($message->toIterable())));
     }
+
+    public function testSerialization()
+    {
+        $message = new RawMessage('string');
+        $this->assertEquals('string', unserialize(serialize($message))->toString());
+        // calling methods more than once work
+        $this->assertEquals('string', unserialize(serialize($message))->toString());
+
+        $message = new RawMessage(new \ArrayObject(['some', ' ', 'string']));
+        $message = new RawMessage($message->toIterable());
+        $this->assertEquals('some string', unserialize(serialize($message))->toString());
```

Comment:
This line will throw `Exception: Serialization of 'Generator' is not allowed` without the fix above